client/src/components/QuoteOfTheDay/QuoteService.test.js

```diff
@@ -0,0 +1,71 @@
+import {
+    fetchKanyeQuote,
+    fetchInspirationalQuote,
+    fetchTaylorSwiftQuote,
+    fetchProgrammingQuote,
+    fetchAnimeQuote,
+} from "./quoteService";
+
+let text;
+let author;
+function setText(newText) {
+    text = newText;
+}
+function setAuthor(newAuthor) {
+    author = newAuthor;
+}
+
+beforeEach(() => {
+    text = "";
+    author = "";
+});
+
+test("It fetches Kanye West quotes correctly", () => {
```

Comment:
Everything looks good, but I'm not sure if these tests are working properly. We tried to follow this way of testing for the weather widget but could never get them to fail, so I ran your tests with a different, incorrect value on line 29 and it doesn't fail either.
We were able to get ours working by adding adding a return where the method is called, so for this one at the start of line 24